src/menu.tsx

```diff
@@ -394,7 +394,7 @@ export const Menu: React.FC<MenuProps> = ({
         <div {...getPrefixedDataAttributes(dataAttributes, 'Menu')}>
             {renderTarget({...targetProps, isMenuOpen})}
 
-            <Portal>
+            <Portal className={styles.wrapper}>
```

Comment:
I think it would be better if `Portal` adds this style by default. This way the isolation feature comes for free and you don't need to set the style in all usages

```
// portal.tsx
// ...
const newContainer = document.createElement('div');
// the style could be added here
```

Reply:
done